src/dust/params.yaml

```diff
@@ -12,10 +12,45 @@ dust:
     _description: "Density floor"
     _units: "g cm^-3"
 
+  grain_density:
+    _type: Real
+    _default: "1.0"
+    _description: "Grain density."
+    _units: "g cm^-3"
+
   nspecies:
     _type: int
     _default: 1
     _description: "The number of dust species"
+  
+  size_input:
+    _type: string
+    _default: direct
+    _description: "How to specify the dust sizes"
+    direct:
+      _type: opt
+      _description: "Directly specify the size distribution as a comma separated list"
+    linspace:
+      _type: opt
+      _description: "Grains are distributed uniformly from a minimum size to a maximum size."
+    logspace:
+      _type: opt
+      _description: "Grains are distributed uniformly in log space from a minimum size to a maximum size."
+    file:
+      _type: opt
+      _description: "Grains are provided in a specified input file."
+
+  min_size:
+    _type: Real
+    _description: "Minimum dust size. For use with size_input=linspace or logspace."
+    _units: "cm"
+  max_size:
+    _type: Real
+    _description: "Maximum dust size. For use with size_input=linspace or logspace."
+    _units: "cm"
+  size_file:
```

Comment:
Should this be immediately preceded by a line break? I'm not sure I understand our convention for when to have line breaks between sets of parameters.

Reply:
I don't know if there's a convention. I can add the line break